src/pages/workspace/members/WorkspaceMemberNewCardPage.tsx

```diff
@@ -99,6 +99,9 @@ function WorkspaceMemberNewCardPage({route, personalDetails}: WorkspaceMemberNew
 
     const handleSelectFeed = (feed: CardFeedListItem) => {
         setSelectedFeed(feed.value);
+        if (isCustomFeed(feed.value as CompanyCardFeed)) {
```

Comment:
how does this look like if you just really fast clicking on the row and clicking next? do we provide some feeds? I feel like we might want to show a loading spinner on the next button if you online and wait for the response so we dont clicker the no cards to assign page

Reply:
ok, makes sense, I will add this